CHANGES.md

```diff
@@ -1,9 +1,23 @@
 # Changes
 
+## edge-20.10.6
+
+This edge supersedes edge-20.10.5 as a release candidate for stable-2.9.0. It
+adds a new `linkerd.io/inject: ingress` annotation to support per-request
+routing when injecting ingress controllers.
+
+* Added a new 'ingress mode' configuration to the proxy to support per-request
+  routing for HTTP ingress controllers
+* Added support for the `linkerd.io/inject: ingress` annotation to configure the
+  proxy in ingress mode
```

Comment:
We could probably drop the first point about `ingress mode` and only mention the annotation, or combine them... It's only one change from the user's point of view